src/rpc/mining.cpp

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) 2010 Satoshi Nakamoto
 // Copyright (c) 2009-2016 The Bitcoin Core developers
-// Copyright (c) 2021 The Dogecoin Core developers
+// Copyright (c) 2023 The Dogecoin Core developers
```

Comment:
If you *really* want to change the copyright line, please use a range when you edit it. In this case it should be 2021-2023.

PS: I have a [script](https://gist.github.com/patricklodder/210a449896ece9ca4a8f872328198e3f) for that too, so you don't have to bother really - sorry.

Reply:
No need to close because you can make a single commit from all these. That's a good skill to learn.

See https://github.com/dogecoin/dogecoin/blob/master/CONTRIBUTING.md#squashing-commits